### DISCUSSION TITLE
Minimal bash-like line

### DISCUSSION THREAD

**Discussion by mvasiliiv** [ventoy/Ventoy#1244] [Q&A] [answered]
Good afternoon, I ran into such a problem when downloading from Ventoy (on the screenshot) 
http://lostpix.com/img/2021-11/15/el8syn1ezgp7aymh9f6es94qd.jpg
on the old PC, this screen I see immediately after selecting a flash drive in the menu Boot, it was not possible to move on, it is due to the fact that my ventoy flash drive So should there be any solution? I do not have any access to the computer, but for the future I would like to know with such situations, thanks

**Comment by ventoy** [comment, marked as answer]
Please refer the FAQ:  https://www.ventoy.net/en/faq.html
`6. Ventoy boot into grub shell`

**Reply by mvasiliiv** [reply]
Understood thanks

**Comment by mvasiliiv** [comment]
Please tell me how to do it correctly, in the screenshot number 1 or number 2
http://lostpix.com/img/2021-11/21/dnoftqgx27j57p0f8yq1icdcp.png
http://lostpix.com/img/2021-11/21/fkmoy1t4ka838hx4w43urjl1l.png

**Reply by ventoy** [reply]
number 2

**Reply by mvasiliiv** [reply]
So the method did not work, today I again encountered such a computer where the old bios does not see more than 137Gb usb flash drive, this file ventoy.disk.img.xz was in the right directory, but nothing happened. I didn't format it in ntfs, I left it in exfat as it was after the ventoy recording, is that the reason, or did I misunderstand something again?

**Reply by ventoy** [reply]
Different motherboard BIOS have different limitation values. Some HP server even doesn't see more than 8GB!
So I recommend to use workaround 2 in https://www.ventoy.net/en/doc_legacy_limit.html

**Reply by mvasiliiv** [reply]
Do I understand correctly that I need to manually format the first large exfat partition and transfer all the necessary folders and files to it again, including ventoy.disk.img.xz, and after that the bios restriction will stop interfering with normal operation?

Won't I ruin any hidden loaders on this section with manual formatting?

**Reply by mvasiliiv** [reply]
Please answer the last two questions

**Comment by mvasiliiv** [comment]
I did everything according to the instructions, formatted it manually, transferred the files back, but the error remained, the workaround did not help

**Comment by ventoy** [comment]
There are two workarounds in https://www.ventoy.net/en/doc_legacy_limit.html
Please use another workaround. 
***You need to reinstall Ventoy and preserve some space at the end of the USB when you reinstall.***

Workaround 1 (Recommend)
You can preserve some space at the end of the disk when you install Ventoy. We can make use of this feature, to preserve some space at the end of the disk and to "extrusion" Ventoy's part1 and part2 within the BIOS's access range. Take a 256GB USB drive for example, if the BIOS access limitation is 137GB, we can preserve 150GB space when install Ventoy. So the part1 and part2 of Ventoy will be at the first 106GB space and will not trigger the BIOS's limitation. Please refer [Notes](https://www.ventoy.net/en/doc_disk_layout.html#reserve_space) for detailed information of the feature.

Firstly, you can preserve more space to test. Because you don't known the exactly BIOS access limit value in your computer.
So if you use a 64GB USB, preserve 60GB space at the end of the USB when you reinstall Ventoy. So Ventoy part1 and part2 will be forced to locate at the first 4GB range, and this should work for most BIOS.

**Comment by mvasiliiv** [comment]
I knew about the fact that you can divide the media into several parts, this option does not suit me, so I tried to make it work on one large partition, but if this is impossible, then we close the question

**Reply by ventoy** [reply]
Since the workaround doesn't work in your computer, I think it's impossible to make it work on the old copmuter with one big part.
